### PR TITLE
Updated user photo in navbar after user login.

### LINKED ISSUE #346: To give an option of logout Button in navbar after successful login of a user.
## I explored this website and found a issue in navbar.

I login successfully with my G-mail account in website but then too navbar shows the option to sign in and sign up.
If i would get an option to log-out there that would be more clear to the user to logout successfully from this application.

### PR DESCRIPTION
**Pull request by ritik-vashist**
Fixes #346

Here is the images after change

**Updated user photo with hi,{user} message in the navbar after login**

### CODE REVIEW

**Comment by ritik-vashist** [comment]
@vasudevsinghal @Abhishek-kumar09 Sir, kindly check my PR and share me your valuable feedback about it.

File: src/components/NavBar/index.js [hunk 1]
```diff
@@ -1,200 +1,214 @@
...
+         makeStyles,
+         Toolbar,
+         Typography,
+       } from "@material-ui/core";
+       import React, { useEffect, useState } from "react";
+       import MenuIcon from "@material-ui/icons/Menu";
+       import { Link } from "react-router-dom";
+       import SearchBar from "material-ui-search-bar";
+       import LinearProgress from "@material-ui/core/LinearProgress";
+       import { useSelector } from "react-redux";
```
Comment (Abhishek-kumar09) [inline src/components/NavBar/index.js:17]: Please revert this.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Please revert all the code changes that are not related to this PR. There are many.

File: src/components/NavBar/index.js (hunk 1 above)
Comment (ritik-vashist) [reply src/components/NavBar/index.js:17]: @Abhishek-kumar09 Sir , update the desired file in #352.
Kindly check it once.

**Pull request closed without merging**